lib/marc_format_processor.rb

```diff
@@ -45,32 +45,40 @@ def resolve_leader(record)
 
```

Comment:
I think I'd like to see this method renamed `resolved_leader_byte_6` (is now `resolve_leader`)

I also kinda wonder how it would look to give these methods more human names and hide their machine-code complexity? Like instead of `resolved_leader_byte_6` it could be called `type_of_record` and then `resolve_leader_byte_7 ` could be `bibliographic_level`. Cribbing from https://www.loc.gov/marc/bibliographic/bdleader.html

I wonder if that'd be easier to read and understand. I kinda like the idea of using method names to help label what the marc data is trying to discern about the record..

Reply:
sounds good to me, changing them

Reply:
I do really like this better 👍 Thanks

one small thing:

`holdings_formats`... kinda not getting this one sorry. `949t` seems like "special local possibilities for format." Is that right? Like these are specially defined by penn state formats. Maybe `local_formats` might be better? Or something?